ludwig/utils/triton_utils.py

```diff
@@ -539,13 +545,15 @@ def generate_scripted_module(self):
 
 
 def export_triton(
-    model: LudwigModel,
-    data_example: pd.DataFrame,
-    output_path: str = "model_repository",
-    model_name: str = "ludwig_model",
-    model_version: Union[int, str] = 1,
-    device: str = "cpu",
-    device_count: int = 1,
+        model: LudwigModel,
+        data_example: pd.DataFrame,
+        predictor_max_batch_size: int = 1,
```

Comment:
`nit` can we move these to the end, given they are probably the most "optional" args, and if folks want to include all inputs without naming them, they could omit these.

Reply:
Which ones should be moved to the end/what order you think would make most sense?

Reply:
I think just the two new properties you added for `max_batch_size` and `max_queue_delay_microseconds` could go after device count.